afnexus/raid-website
# chore(ci): Add husky pre commit lint and format

## Description
- Add prettier
- Add husky pre-commit hook that runs prettier and next lint
- Add formatting changes

## Steps to test
1. Make any change
2. Run `git add .`
3. Run `git commit -m "test commit"`
4. Pre-commit hook should run

## Screenshots / Videos

| Screenshot |
| ---------- |
| ![image](https://user-images.githubusercontent.com/13270108/208285151-da745595-b432-4f7e-b546-9fdff0b39c11.png) |

CLOSES #19
